components/TextTooltip.tsx

```diff
@@ -4,15 +4,25 @@ import { Tooltip } from 'react-tooltip';
 
 type Props = {
   customStyle?: string;
+  isOpen?: boolean;
 };
 
-const TextTooltip = ({ customStyle }: Props) => (
-  <div className='text-tooltip'>
-    <Tooltip
-      id='text-tooltip'
-      className={classNames('text-tooltip-body', customStyle)}
-    />
-  </div>
-);
+const TextTooltip = ({ customStyle, isOpen }: Props) => {
+
+  return (
+    <div
+      className='text-tooltip'
+    >
+      <Tooltip
+        id='text-tooltip'
+        className={classNames('text-tooltip-body', customStyle)}
+        // closeOnScroll is part of V5 API, types 4.x.x don't include this parameter
```

Comment:
We need to update dependencies, it's probably a configuration error.

1. @types/react-tooltip should be removed. It's a deprecated package, types are now included in react-tooltip package https://www.npmjs.com/package/@types/react-tooltip https://github.com/ReactTooltip/react-tooltip/blob/master/src/components/Tooltip/TooltipTypes.d.ts
2. closeOnScroll was added in 5.14.0 which is included because of caret usage in our package.json but it should be updated to at least "^5.14.0" to explicitly show that this is a minimum version for the application to function as it should https://github.com/ReactTooltip/react-tooltip/releases/tag/v5.14.0

Reply:
@dszafranek Thanks for the tip! Updating the library actually resolved the issue without need to do a workaround with scroll event, which much simplified code changes. Can you re-review?